lib/util/plist.js

```diff
@@ -0,0 +1,56 @@
+import bplistParser from 'bplist-parser';
+import bplistCreator from 'bplist-creator';
+import plist from 'plist';
+
+const BPLIST_IDENTIFIER = {
+  BUFFER: Buffer.from('bplist00'),
+  TEXT: 'bplist00'
+};
+const PLIST_IDENTIFIER = {
+  BUFFER: Buffer.from('<'),
+  TEXT: '<'
+};
+
+function getTextPlist (data) {
+  if (typeof data === 'string' && data.startsWith(PLIST_IDENTIFIER.TEXT)) {
```

Comment:
use lodash _.isString

Reply:
one could also use _.startsWith lodash method, which properly handles invalid/null/undefined arguments

Reply:
Cool. Will check that out